packages/reactstrap-validation-date/src/AvDateRange.js

```diff
@@ -296,7 +343,10 @@ export default class AvDateRange extends Component {
       this.props.end.name
     ).getViewValue();
 
-    if (start && end) {
+    const hasStart = start && start !== '';
```

Comment:
empty strings are falsy so the second condition will never get hit. i think line 349 can just be:
```js
if (start && end)
```

Reply:
Hmmm, for some reason though if you didn't do this and you ran the build on the `docs`. It would actually not work properly and the validation would fail.

Steps to replicate:
- checkout ee00a6b4e04de95f3c9e377625f91dbc2b4691cd
- `npm run build:docs`
- open `index.html` file in the `docs` build folder at the root

Reply:
This is wild.

Transpiled code for `requireStartIfEnd invalid format:
```
AvDateRange_defineProperty(AvDateRange_assertThisInitialized(_this), "requireStartIfEnd", function() {
                    console.log("require start if end");
                    var start = _this.context.FormCtrl.getInput(_this.props.start.name).getViewValue()
                      , end = _this.context.FormCtrl.getInput(_this.props.end.name) && 
                       _this.context.FormCtrl.getInput(_this.props.end.name).getViewValue();
                    return start || !end || "Both start and end date are required."
  }),
```

Reply:
It must assume that things which are "not"ed are booleans 🤷‍♂.